bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/handler/EnOceanBaseSensorHandler.java

```diff
@@ -202,9 +202,9 @@ public void packetReceived(BasePacket packet) {
                 // fire trigger for receive
                 triggerChannel(prepareAnswer, "requestAnswer");
                 // Send response after 100ms
-                ScheduledFuture<?> localResponseFuture = responseFuture;
-                if (localResponseFuture == null || localResponseFuture.isDone()) {
-                    localResponseFuture = scheduler.schedule(this::sendRequestResponse, 100, TimeUnit.MILLISECONDS);
+                ScheduledFuture<?> responseFuture = this.responseFuture;
+                if (responseFuture == null || responseFuture.isDone()) {
+                    this.responseFuture = scheduler.schedule(this::sendRequestResponse, 100, TimeUnit.MILLISECONDS);
```

Comment:
Please revert to as it was, only add a line (208):
```suggestion
                    this.responseFuture = localResponseFuture
```

Reply:
See above.

Reply:
See other comments.